src/metabase/api/session.clj

```diff
@@ -191,16 +191,21 @@
         (throw (ex-info (tru "Email is not verified.") {:status-code 400}))))))
 
 ;; TODO - are these general enough to move to `metabase.util`?
+(defn- split-domains [domains]
+  (apply vector
+    (map clojure.string/trim
+      (clojure.string/split domains #","))))
+
 (defn- email->domain ^String [email]
   (last (re-find #"^.*@(.*$)" email)))
 
-(defn- email-in-domain? ^Boolean [email domain]
+(defn- email-in-domains? ^Boolean [email domains]
   {:pre [(u/email? email)]}
-  (= (email->domain email) domain))
+  (some #(= (email->domain email) %) (split-domains domains)))
```

Comment:
I would just do this like

```clojure
(contains? (set (str/split domains-str #"\s*,\s*")) (email->domain email))
```

*  `require` `metabase.string :as str` and use that. See https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#require-namespaces-you-use
*  You should probably name the param for this function `domains-str` or something so people don't get confused and think it's a sequence of domains